eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -364,19 +311,8 @@ function New-MgmtPackageFolder() {
       $mgmtPackageName = $folderinfo.Name
       $projectFolder = "$sdkPath/sdk/$packageName/$mgmtPackageName"
     } else {
-      Write-Host "Path doesn't exist. create template."
-      dotnet new -i $sdkPath/eng/templates/Azure.ResourceManager.Template
-      $CaptizedPackageName = [System.Globalization.CultureInfo]::InvariantCulture.TextInfo.ToTitleCase($packageName)
-      $mgmtPackageName = "Azure.ResourceManager.$CaptizedPackageName"
-      $projectFolder="$sdkPath/sdk/$packageName/Azure.ResourceManager.$CaptizedPackageName"
-      Write-Host "Create project folder $projectFolder"
-      New-Item -Path $projectFolder -ItemType Directory
-      Push-Location $projectFolder
-      dotnet new azuremgmt --provider $packageName --includeCI true --force
-      Pop-Location
-
-      $packagesPath = Join-Path $sdkPath "sdk"
-      RegisterMgmtSDKToMgmtCoreClient -packagesPath $packagesPath
+      Write-Error "Project directory doesn't exist. create template."
```

Comment:
How are the users supposed to create templates?

Reply:
Change the error message, and add detail log to tell customer that we will not allowed to generate new sdk from swagger.